### PR TITLE
Feature robot window on startup for stream

### PR DESCRIPTION
**Pull request by Jean-Eudes-le-retour**
**Description**
This PR makes the robot window visible on the streaming client on startup if they have been set to open in the wbproj file.

This would make the web streaming experience more coherent with the local Webots experience. This would be an easier way than #5753 to set robot windows to open on webots.cloud and would make it obsolete. This would also be more flexible, as it would allow people to submit simulations in webots.cloud with robot windows opened by default.

It can be tested locally by calling webots with the --stream argument, opening a robotbenchmark sample world and going to http://localhost:1234/index.html to see the robot windows being visible.

**Tasks**

- [x] replace "../resources/web/wwi" with https://cyberbotics.com/wwi/R2023b before merge

### CODE REVIEW

File: resources/web/wwi/Toolbar.js [hunk 1]
```diff
@@ -553,8 +555,6 @@ export default class Toolbar {
       if (rw.isMainWindow) {
         rw.setSize(2 * robotWindowWidth, 2 * robotWindowHeight);
         rw.setPosition(margin, margin);
-        if (this.parentNode?.openMainRobotWindow === true)
-          rw.changeVisibility();
```
Comment (BenjaminDeleze) [inline resources/web/wwi/Toolbar.js:557]: You removed what was introduced in #5753 right? In this case, we should also remove the corresponding documentation

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Seems good, just a question:

File: resources/web/wwi/Toolbar.js (hunk 1 above)
Comment (Jean-Eudes-le-retour) [reply resources/web/wwi/Toolbar.js:557]: Oh yeah, right! I'll do that

**Comment by Jean-Eudes-le-retour** [comment]
I left some comments in the code for me to understand how some part of the code worked, should I leave them or remove them ?

**Review by BenjaminDeleze — APPROVED** [review]
Thank you
Comments are fine for me

**Pull request merged**